ambry-store/src/test/java/com.github.ambry.store/IndexSegmentUtils.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.store;
+
+import com.github.ambry.utils.CrcOutputStream;
+import java.io.DataOutputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.Map;
+
+
+/**
+ * IndexSegment Utils to assist in persisting the {@link IndexSegment} in version {@link PersistentIndex#VERSION_0}
+ */
+class IndexSegmentUtils {
```

Comment:
test utils sounds more appropriate and not sure if it needs a class file of its own. Seems like this can be a static function inside of `IndexSegmentTest`. Not for introducing classes for one or two static functions that have limited utility.